scripts/SCR_GAMESTART/SCR_GAMESTART.gml

```diff
@@ -94,7 +94,7 @@ function SCR_GAMESTART()
 	}
 	global.xpreward[3] = 0
 	global.goldreward[3] = 0
-	global.battlegroup = 3
+	global.battlegroup = BattleGroup.Froggit
```

Comment:
battlegroup 3 according to the macro is supposed to be BattleGroup.TorielFroggit and not BattleGroup.Froggit, this would set it to 4 instead